src/components/EventCard/style.scss

```diff
@@ -93,11 +93,11 @@
       font-weight: 600;
       width: 150px;
       margin: 0 auto;
-      cursor: not-allowed;
+      cursor: pointer;
```

Comment:
like my other comment it would be good to have this be a part of a conditional class. Like 'active' or 'past'